apps/site/src/app/schedule/DayOne/DayOne.module.scss

```diff
@@ -0,0 +1,87 @@
+@use "bootstrap-utils" as bootstrap;
+
+.mapContainer {
+	width: 100vw;
+	position: relative;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height: auto;
+	height: 1300px;
+	@include bootstrap.media-breakpoint-down(md) {
+		height: 1500px;
+	}
+}
+
+.map1 {
+	position: absolute;
+	width: 80%;
+	min-width: 1000px;
+	@include bootstrap.media-breakpoint-down(md) {
+		display: none;
+	}
+}
+
+.title {
+	position: absolute;
+	top: -70px;
+	left: 50%;
+	transform: translateX(-50%);
+}
+
+.mapPath {
+	position: absolute;
+	aspect-ratio: 1 / 1;
+	@include bootstrap.media-breakpoint-down(md) {
+		width: 265px;
+		height: 592px;
+	}
+}
+
+.mobileMap {
+	position: absolute;
+	width: 80%;
+	min-width: 1000px;
+	height: 70%;
+	@include bootstrap.media-breakpoint-up(md) {
+		display: none;
+	}
+}
+
+.mobilePath {
+	position: absolute;
+	height: 100%;
+	width: 100%;
+	@include bootstrap.media-breakpoint-up(md) {
+		display: none;
+	}
+}
+
+.pathImage {
+	width: 100%;
+	height: 100%;
+	@include bootstrap.media-breakpoint-down(md) {
+		display: none;
+	}
+}
+
+@media screen and (min-width: 1501px) {
```

Comment:
suggestion (repeating): these media queries could be replaced with the bootstrap media breakpoints. For example [here](https://github.com/HackAtUCI/zothacks-site/blob/4366ce4c9124b948ace5e79b499a1fa9fc2319f4/apps/site/src/app/(home)/sections/GetInvolved/GetInvolved.module.scss#L19).

Reply:
Although I would like to do this, using fixed bootstrap md, xl, sm, etc. wouldn't mesh well with the already established media queries; it would be nice if there was more time for a bit of cleanup, but for now this should stay